wagtailio/project_styleguide/templates/patterns/includes/search-modal.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal modal--search micromodal-slide" id="site-search-modal" aria-hidden="true">
+    <div class="modal__overlay" tabindex="-1" data-micromodal-close>
+        <div class="modal__container" role="dialog" aria-modal="true">
+            <main class="modal__content">
+                <button class="modal__close-button" aria-label="Close the Search" data-micromodal-close>
+                    <svg class="modal__close-icon" aria-hidden="true">
```

Comment:
Note: doesn't this icon have a purpose that is not just functional? I thought this would not be hidden

Reply:
It has a purpose for users not using assistive tech but for users using it then it doesn't serve a purpose - this is where the aria-label on the parent button will be announced instead.